includes/admin/views/html-admin-page-addons.php

```diff
@@ -29,15 +29,24 @@
 				<li><a class="<?php echo $current_section === $section_id ? 'current' : ''; ?>" href="<?php echo admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section_id ) ); ?>"><?php echo esc_html( $section->title ); ?></a><?php echo ( end( $section_keys ) !== $section_id ) ? ' |' : ''; ?></li>
 			<?php endforeach; ?>
 		</ul>
-		<br class="clear" />
+		<form class="search-form" method="GET">
+			<button type="submit">
+				<span class="dashicons dashicons-search"></span>
+			</button>
+			<input type="text" name="search" value="" placeholder="Enter a search term and press enter">
+			<input type="hidden" name="page" value="wc-addons">
+			<?php $page_section = isset( $_GET['section'] ) ? $_GET['section'] : ''; ?>
+			<?php $page_section = 'featured' === $page_section ? '' : $page_section; ?>
```

Comment:
I think this can be simplified:
```
<?php $page_section = ( isset( $_GET['section'] ) && 'featured' !== $page_section ) ? $_GET['section'] : ''; ?>
```

Reply:
Done